_topic_map.yml

```diff
@@ -751,24 +751,20 @@ Topics:
 #  - Name: Controlling virtual machines states
 #    File: cnv-controlling-vm-states
 #  - Name: Accessing virtual machine consoles
-#    File: cnv-accessing-vm-consoles      
+#    File: cnv-accessing-vm-consoles
 #- Name: Container-native Virtualization Release Notes
 #  Dir: cnv_release_notes
 #  Topics:
 #  - Name: CNV Release Notes Placeholder
 #    File: cnv-release-notes-placeholder
 #---
-#Name: Service Mesh
-#Dir: service_mesh
-#Distros: openshift-enterprise
-#Topics:
-#- Name: Service Mesh Installation
-#  Dir: service_mesh_install
-#  Topics:
-#  - Name: Service Mesh Insatll Assemblies Placeholder
-#    File: service-mesh-install-placeholder
-#- Name: Service Mesh Release Notes
-#  Dir: service_mesh_release_notes
-#  Topics:
-#  - Name: Service Mesh Release Notes Placeholder
-#    File: service-mesh-release-notes-placeholder
+Name: Service Mesh
```

Comment:
@JStickler - leave these commented out. If this gets merged, this will display in the live docs before we GA.

Reply:
That was the plan, once I'd seen how it looked in the preview.